create_mountaineer_app/create_mountaineer_app/templates/.gitignore

```diff
@@ -0,0 +1,180 @@
+# General excludes
```

Comment:
Looks like this is based on the github python base?
https://github.com/github/gitignore/blob/main/Python.gitignore

If so, mind dropping a link to the original in the gitignore header so users are aware of where this black magic config came from?

Reply:
There was a comment and link to the source already a bit further down, where the black magic starts. But I preferred to keep the project specific ignores at the top.